Although not thought for the port, this patch was submitted to the varkon 
lists so hopefully we will not need it for the next release. --pfg

*** Makefile.FreeBSD.orig	Wed Apr 16 18:02:04 2003
--- Makefile.FreeBSD	Wed Apr 16 18:23:37 2003
***************
*** 9,15 ****
  #
  ##########################################################
  
! # The varkon object libraies
  
  LIBS  = $(VARKON_ROOT)/sources/IG/lib/IGlib.a \
          $(VARKON_ROOT)/sources/PM/lib/PMlib.a \
--- 9,15 ----
  #
  ##########################################################
  
! # The varkon object libraries
  
  LIBS  = $(VARKON_ROOT)/sources/IG/lib/IGlib.a \
          $(VARKON_ROOT)/sources/PM/lib/PMlib.a \
***************
*** 21,31 ****
  
  # Xlibs 
  
! XLIBS = -lX11
  
  # OpenGL libs (currently not supported on Sun's)
  
! GLIBS = -IGL -IGLU
  
  # The actual executable, xvarkon.
  
--- 21,31 ----
  
  # Xlibs 
  
! XLIBS = -L/usr/X11R6/lib -lX11 -lXext
  
  # OpenGL libs (currently not supported on Sun's)
  
! GLIBS = -lGL -lGLU
  
  # The actual executable, xvarkon.
  
***************
*** 34,40 ****
  # Targets
  
  $(XVARKON): $(LIBS)
! 	cc $(LIBS) $(XLIBS) -lm -o $(XVARKON)
  
  
  ##########################################################
--- 34,40 ----
  # Targets
  
  $(XVARKON): $(LIBS)
! 	cc $(LIBS) $(XLIBS) $(GLIBS) -lm -o $(XVARKON)
  
  
  ##########################################################
*** IG/src/ig2.c.orig	Sat Apr 12 16:55:19 2003
--- IG/src/ig2.c	Sat Apr 12 16:55:45 2003
***************
*** 54,60 ****
  
  #ifdef UNIX
  #undef VSTART
! #include "termio.h"
  #endif
  
  extern MNUALT smbind[];      /* Symboler */
--- 54,60 ----
  
  #ifdef UNIX
  #undef VSTART
! #include "termios.h"
  #endif
  
  extern MNUALT smbind[];      /* Symboler */
*** PL/src/epson.c.orig	Tue Apr 15 10:31:15 2003
--- PL/src/epson.c	Tue Apr 15 10:33:10 2003
***************
*** 44,50 ****
  #include "../include/params.h"
  #include <string.h>
  #include <time.h>
! #include <malloc.h>
  
  /*
  *******************************************************!*/
--- 44,50 ----
  #include "../include/params.h"
  #include <string.h>
  #include <time.h>
! #include <stdlib.h>
  
  /*
  *******************************************************!*/
*** PL/src/gerber.c.orig	Tue Apr 15 12:17:21 2003
--- PL/src/gerber.c	Tue Apr 15 12:17:40 2003
***************
*** 44,50 ****
  #include "../include/params.h"
  #include <string.h>
  #include <time.h>
! #include <malloc.h>
  
  #define ppixsz   0.0254 /* Plotter pixel x-size i mm */
  #define ppiysz   0.0254 /* Plotter pixel y-size i mm */
--- 44,50 ----
  #include "../include/params.h"
  #include <string.h>
  #include <time.h>
! #include <stdlib.h>
  
  #define ppixsz   0.0254 /* Plotter pixel x-size i mm */
  #define ppiysz   0.0254 /* Plotter pixel y-size i mm */
*** PL/src/ibmpro.c.orig	Tue Apr 15 12:16:13 2003
--- PL/src/ibmpro.c	Tue Apr 15 12:16:48 2003
***************
*** 44,50 ****
  #include "../include/params.h"
  #include <string.h>
  #include <time.h>
! #include <malloc.h>
  
  bool   arccon;          /* Måla cirklars rand */
  short  lastx;           /* Sista pos X */
--- 44,50 ----
  #include "../include/params.h"
  #include <string.h>
  #include <time.h>
! #include <stdlib.h>
  
  bool   arccon;          /* Måla cirklars rand */
  short  lastx;           /* Sista pos X */
*** PM/src/Makefile.FreeBSD.orig	Tue Apr 15 10:23:40 2003
--- PM/src/Makefile.FreeBSD	Tue Apr 15 10:24:38 2003
***************
*** 19,25 ****
              -O                          \
              -DWRK_STAT                  \
              -DV3_X11 -I/usr/X11R6/include \
!             -DUNIX
  
  debug :      CFLAGS += -g -DDEBUG
  
--- 19,25 ----
              -O                          \
              -DWRK_STAT                  \
              -DV3_X11 -I/usr/X11R6/include \
!             -DUNIX -DVARKON
  
  debug :      CFLAGS += -g -DDEBUG
  
*** AN/src/Makefile.FreeBSD.orig	Tue Apr 15 10:26:37 2003
--- AN/src/Makefile.FreeBSD	Tue Apr 15 10:27:28 2003
***************
*** 28,34 ****
           -D_POSIX_SOURCE \
           -DUNIX \
           -DANALYZER \
!          -DWRK_STAT -I/usr/X11R6/include
  
  LDFLAGS = -L/usr/X11R6/lib
  
--- 28,35 ----
           -D_POSIX_SOURCE \
           -DUNIX \
           -DANALYZER \
!          -DWRK_STAT -I/usr/X11R6/include \
!          -DVARKON
  
  LDFLAGS = -L/usr/X11R6/lib
  
